region/tourian/main/Tourian Escape Room 4.json

```diff
@@ -411,6 +411,25 @@
         "Alternatively, jump and force the pirates to shoot just before starting to climb."
       ]
     },
+    {
+      "link": [3, 1],
+      "name": "IBJ and Acid Bath",
+      "requires": [
+        "canSuitlessLavaDive",
+        "canBePatient",
+        "canJumpIntoIBJ",
+        "canDoubleBombJump",
+        {"or": [
+          {"and": [
+            "Gravity",
+            {"acidFrames": 150}
+          ]},
+          {"acidFrames": 450}
+        ]}
+      ],
+      "note": "Clear the right side Space Pirates and then race the rising acid by bomb jumping.",
+      "devNote": "canBePatient mostly for difficulty placement."
```

Comment:
```suggestion
      "note": "Clear the right side Space Pirates and then race the rising acid by bomb jumping."
```
I think using patience tech for difficulty placement may not be ideal (unless the strat actually fits the description of the patience tech). There's a possibility that in the future we may want to pull out the patience tech into a different place in the randomizer options separate from other tech assumptions, so that's one reason to try to keep its meaning consistent.